### PR TITLE
feat: add rendered manifest on tags and push on PRs

### PR DESCRIPTION
**Pull request by maxres-fr**

### CODE REVIEW

File: cloudbuild.yaml [hunk 1]
```diff
@@ -14,6 +14,8 @@ steps:
     env: ['GOPATH=/workspace/gopath']
     args: ['-c', 'cd ForgeRock/ds-operator && make build PR_NUMBER=$_PR_NUMBER']
     secretEnv: ['GITHUB_TOKEN']
+images:
+  - 'gcr.io/$PROJECT_ID/ds-operator:pr-$_PR_NUMBER'
```
Comment (jrcast) [inline cloudbuild.yaml:18]: $PROJECT_ID here? I see `forgeops-public` above

**Review by jrcast — APPROVED** [review]

File: cloudbuild.release.yaml [hunk 2]
```diff
@@ -13,7 +13,7 @@ steps:
   dir: gopath/src/github.com
   env: ['GOPATH=/workspace/gopath']
   # we "create" the tag because cloud build doesn't pull the tags, just commit
-  args: ['-c', 'cd ForgeRock/ds-operator && git tag $TAG_NAME && IMG=gcr.io/$PROJECT_ID/ds-operator:$TAG_NAME make release' ]
+  args: ['-c', 'cd ForgeRock/ds-operator && git tag $TAG_NAME && IMG=gcr.io/forgeops-public/ds-operator:$TAG_NAME make release' ]
```
Comment (jrcast) [inline cloudbuild.release.yaml:16]: here

Comment (wstrange) [reply cloudbuild.release.yaml:16]: I think the challenge is we want pre-release to go to engineering-devops, and releases to go to forgeops-public.

**Review by wstrange — APPROVED** [review]

File: cloudbuild.release.yaml (hunk 2 above)
Comment (jrcast) [reply cloudbuild.release.yaml:16]: I remember now 👍

**Pull request closed without merging**